roles/contiv_network/tasks/fwd_tasks.yml

```diff
@@ -0,0 +1,3 @@
+- name: set fwd mode
+  shell: netctl global set --fwd-mode "{{ fwd_mode }}" 
+  run_once: true
```

Comment:
@abhinandanpb `run_once` will cause the task to be run only on one node among all nodes that are present in a host-group at the time ansible is run. Is that what you want?